### DISCUSSION TITLE
Logout Help

### DISCUSSION THREAD

**Discussion by talha1230** [community/community#137859] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hey Copilots,
I recently logged in my GitHub Copilot in my friends vscode, in more then three laptops, now I'm having difficulty and clashing with my own vs-code Copilot, how do i logout my Copilot account from their laptop?

**Comment by MillorJ** [comment, marked as answer]
To log out of GitHub Copilot on the other laptops where you signed in, you can follow these steps to revoke access from your GitHub account and also remove the credentials stored on those devices:

1. Revoke Authorized OAuth Apps on GitHub:
You can revoke the authorization of GitHub Copilot from your GitHub account, which will force the other laptops to be signed out.
-Go to GitHub > Settings.
-Navigate to Applications under Authorized OAuth Apps.
-Find GitHub Copilot in the list of authorized applications.
-Click Revoke Access for GitHub Copilot.
This will sign out GitHub Copilot on all devices where you’ve logged in, including your friend's laptops.

2. Remove Credentials from VS Code:
If you have access to those laptops, you can manually sign out from GitHub Copilot in VS Code.
-Open VS Code on the other laptops.
-Press F1 (or Ctrl+Shift+P) to open the Command Palette.
-Type GitHub: Sign Out and select it. This will log out your account from VS Code on that machine.

3. Change GitHub Account Password (Optional):
If you want to ensure no further access to your account, you can change your GitHub password, which will also invalidate any saved sessions.

Following these steps should ensure that your Copilot account is fully logged out from the other laptops.
